core/bind/core_bind.cpp

```diff
@@ -1292,6 +1297,8 @@ void _OS::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("center_window"), &_OS::center_window);
 	ClassDB::bind_method(D_METHOD("move_window_to_foreground"), &_OS::move_window_to_foreground);
 
+	ClassDB::bind_method(D_METHOD("get_native_handle"), &_OS::get_native_handle);
```

Comment:
You should give the argument names too (that's why currently you have `arg0` and `arg1` in the doc).

Reply:
```suggestion
	ClassDB::bind_method(D_METHOD("get_native_handle", "handle_type", "index"), &_OS::get_native_handle);
```

Reply:
Ah! I was wondering about that, too used to not doing that in GDNative plugins